apis/stardust/api-spec/spec.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "name": "@nebula.js/stardust",
     "description": "Product and framework agnostic integration API for Qlik's Associative Engine",
-    "version": "1.2.0",
+    "version": "1.7.0",
```

Comment:
Did we miss some steps?

Reply:
we missed updating the specs version yea